HAPNodeJSClient.js

```diff
@@ -539,18 +576,19 @@ HAPNodeJSClient.prototype.HAPresource = function(ipAddress, port, body, callback
  * HAPNodeJSClient.prototype.HAPstatusByDeviceID - Get current status for characteristics
  *
  * @param  {type} deviceID  deviceID of homebridge instance
+ * @param  {type} pin       Pin of homebridge instance
  * @param  {type} body      description
  * @param  {type} callback  Callback to execute upon completion of characteristic getting, function(err, response)
  */
 
-HAPNodeJSClient.prototype.HAPstatusByDeviceID = function(deviceID, body, callback) {
+HAPNodeJSClient.prototype.HAPstatusByDeviceID = function(deviceID, pin, body, callback) {
```

Comment:
This is a breaking change to the external API and can not be merged as it will break all other packages using this package